README.md

```diff
@@ -4,10 +4,16 @@
   />
 </p>
 <p align="center">
-<a href="https://travis-ci.com/psincraian/pepy"><img src="https://travis-ci.com/psincraian/pepy.svg?branch=master" alt="Build status" height="18"></a>
+
+[![Build Status](https://travis-ci.com/psincraian/pepy.svg?branch=master)](https://travis-ci.com/psincraian/pepy)
+![PyPI - Status](https://img.shields.io/pypi/status/pype)
+![PyPI - License](https://img.shields.io/pypi/l/pype)
```

Comment:
the project name is pepy and it's in the PyPI repository. So if you delete these two comments I will apply the changes 😊 

```suggestion
```